storm-server/src/test/java/org/apache/storm/utils/ServerUtilsTest.java

```diff
@@ -272,18 +290,28 @@ public void testIsAnyPosixProcessPidDirAliveMockingFileOwnerUid() throws Excepti
             LOG.info("Test testIsAnyPosixProcessPidDirAlive is designed to run on systems with /proc directory only, marking as success");
             return;
         }
-        Collection<Long> pids = getRunningProcessIds();
-        assertFalse(pids.isEmpty());
+        Collection<Long> allPids = getRunningProcessIds(null);
```

Comment:
It doesn't seem very necessary to get all pids. We just need the pids from currentUser (`System.getProperty("user.name")`)

Reply:
This tests the case where some (but not all) PIDs have been reassigned.